agent/src/agent.ts

```diff
@@ -403,11 +403,14 @@ export class Agent extends MessageHandler implements ExtensionClient {
                 this.workspace.loadDocumentWithChanges(documentWithUri)
             const textEditor = this.workspace.newTextEditor(textDocument)
             this.workspace.setActiveTextEditor(textEditor)
-            vscode_shim.onDidChangeTextDocument.fire({
-                document: textDocument,
-                contentChanges,
-                reason: undefined,
-            })
+
+            if (contentChanges) {
```

Comment:
This will always be true even if contentChanges is empty right? Cause `Boolean([])` returns `true` ?
Will need to update to:

```suggestion
            if (contentChanges?.length) {
```

Reply:
ha, I was just opening this PR to make the same comment, will fix in a followup PR

Reply:
haha It's been fixed @olafurpg! https://github.com/sourcegraph/cody/pull/4278

Reply:
https://github.com/sourcegraph/cody/pull/4279